libs/utils/src/http/error.rs

```diff
@@ -20,6 +20,9 @@ pub enum ApiError {
     #[error("Conflict: {0}")]
     Conflict(String),
 
+    #[error("Precondition failed: {0}")]
+    PreconditionFailed(String),
```

Comment:
With `String` it becomes tempting to do `format!("Tenant {id} is missing")`, with `&'static str` we could work towards more consistent messages.